llama_stack/apis/tools/rag_tool.py

```diff
@@ -85,7 +99,7 @@ class RAGQueryConfig(BaseModel):
     max_tokens_in_context: int = 4096
     max_chunks: int = 5
     chunk_template: str = "Result {index}\nContent: {chunk.content}\nMetadata: {metadata}\n"
-    mode: str | None = None
+    mode: RAGSearchMode = RAGSearchMode.VECTOR
```

Comment:
Note that this makes the `mode` parameter required - for example see the generated specs where `mode` is added to the required list. You likely meant for this to be `mode: RAGSearchMode | None = RAGSearchMode.VECTOR` or something similar.